llrt_utils/src/bytes.rs

```diff
@@ -1,8 +1,247 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Nit space, might want a linter rule for imports. They are available in unstable fmt